styles.css

```diff
@@ -454,105 +470,122 @@ form {
   padding-bottom: 40%;
 }
 
-.toolbar-links {
+.toolbar-links,
+.image-desktop,
+.about-project-desktop,
+.client-name-desktop,
+.name-desktop,
+.li-desktop,
+.project-desktop,
+.menu-logo {
   display: none;
 }
 
-.image-desktop {
-  display: none;
-}
+@media only screen and (min-width: 768px) {
+  /* animation and transition */
+  img {
+    transition: rotate 1s;
+  }
 
-.about-project-desktop {
-  display: none;
-}
+  img:hover {
+    rotate: 360deg;
+  }
 
-.client-name-desktop {
-  display: none;
-}
+  /* headliner page */
+  .toolbar {
+    width: 100vw;
+    height: 3.4vh;
+    position: relative;
+    justify-content: space-around;
+    animation: change-color 5s;
+  }
 
-.name-desktop {
-  display: none;
-}
+  @keyframes change-color {
+    0% {
+      background-color: #fff;
+    }
 
-.li-desktop {
-  display: none;
-}
+    50% {
+      background-color: #7f8cff69;
+    }
 
-.project-desktop {
-  display: none;
-}
+    100% {
+      background-color: #fff;
+    }
+  }
 
-@media (min-width: 768px) {
-  /* headliner page */
   .toolbar-links {
+    font-size: 0.6vw;
     color: #344563;
     text-decoration: none;
     display: flex;
-    gap: 30%;
-    margin-right: 16%;
+    justify-content: start;
+    width: 10vw;
+    gap: 4%;
   }
 
   .toolbar-links > a {
-    padding: 0;
+    font-size: 1vw;
     margin: 0;
-    position: relative;
-  }
```

Comment:
- [ ] What the two previous reviewers are talking about as regards to the line in your header is the underline on the anchor tag links. This is a default text-decoration added by HTML. Kindly remove it using `text-decoration: none;`
![image](https://github.com/agneta1991/Portfolio/assets/94551484/6faf0dcc-8e3b-41ef-a969-991617ebda86)

So sorry for the misunderstanding. Don't forget to ask questions if you don't understand.